### ISSUE TITLE
Add option to use rdfs:label in generated context

### ISSUE THREAD

**Issue by cmungall** [stain/owl2jsonld#5]
In ontologies that use a numeric fragment on their class URIs, it would be useful to use a property like rdfs:label to make the context

**Comment by stain** [comment]
Would `skos:prefLabel` make more sense? Ideally it should be a "code name" with no spaces. But I think this issue is generally for an ontology to be able to provide an alternative to their fragment to use as JSON-LD name.

**Comment by cmungall** [comment]
Yes, generally configurable is the idea. (FWIW, most bio-ontologies use rdfs:label)
